src/core/opamProcess.mli

```diff
@@ -42,6 +42,12 @@ val string_of_command: command -> string
 val text_of_command: command -> string option
 val is_verbose_command: command -> bool
 
+val command_cmd : command -> string
+val command_args : command -> string list
+val command_dir : command -> string
+val command_env : command -> string array
+val command_allow_stdin : command -> bool option
```

Comment:
```suggestion
val command_cmd : command -> string
val command_args : command -> string list
val command_dir : command -> string option
val command_env : command -> string array option
val command_allow_stdin : command -> bool option
```

Reply:
Thanks! I think this was due me vendoring opam and dune apparently just believed the interfaces without building the implementations. Will be careful about this in the future!